2.4/contrib/initiate_replica.sh

```diff
@@ -15,10 +15,7 @@ done
 current_endpoints=$(endpoints)
 mongo_node="$(echo -n ${current_endpoints} | cut -d ' ' -f 1):${CONTAINER_PORT}"
 
-echo "=> Waiting for all endpoints to accept connections..."
-for node in ${current_endpoints}; do
-  wait_for_mongo_up ${node} &>/dev/null
-done
+wait_for_all_hosts
```

Comment:
I think it is redundant to wait_for_all_hosts twice. Several lines after this there is called mongo_initiate and in this function there is also called wait_for_all_hosts.
So I think wait_for_all_hosts in 2.4/contrib/initiate_replica.sh is not necessary.